Neos.Flow/Classes/Core/Booting/Scripts.php

```diff
@@ -356,10 +356,15 @@ public static function initializeCacheManagement(Bootstrap $bootstrap)
         $configurationManager = $bootstrap->getEarlyInstance(ConfigurationManager::class);
         $environment = $bootstrap->getEarlyInstance(Environment::class);
 
-        $cacheFactoryObjectConfiguration = $configurationManager->getConfiguration(ConfigurationManager::CONFIGURATION_TYPE_OBJECTS, CacheFactoryInterface::class);
```

Comment:
Was that always the case? I mean why the hell do we separte the object configuration by package LOL. But that's for another PR / ticket.